tests/lmeval/test_lmeval.py

```diff
@@ -57,6 +59,10 @@ class TestLMEval:
     """  # noqa: E501
 
     def set_up(self):
+        random.seed(42)
+        numpy.random.seed(42)
+        torch.manual_seed(42)
```

Comment:
torch.manual_seed() should be enough?

Reply:
i didn't check, but following the pattern lmeval has in their setup, setting these seeds. should i drop the other 2 and test?

Reply:
I ran with just torch seed set and it failed to reproduce, so leaving all three set